web/index.js

```diff
@@ -101,21 +44,6 @@ export async function createServer(
     res.status(status).send({ success: status === 200, error });
   });
 
-  app.use((req, res, next) => {
-    const shop = shopify.utils.sanitizeShop(req.query.shop);
-    if (shopify.config.isEmbeddedApp && shop) {
-      res.setHeader(
-        "Content-Security-Policy",
-        `frame-ancestors https://${encodeURIComponent(
-          shop
-        )} https://admin.shopify.com;`
-      );
-    } else {
-      res.setHeader("Content-Security-Policy", `frame-ancestors 'none';`);
-    }
-    next();
-  });
-
   if (isProd) {
```

Comment:
We can probably get rid of the compression usage here, and might be able just serve static for both prod and dev.